x/superfluid/keeper/keeper_test.go

```diff
@@ -27,6 +27,7 @@ import (
 
 type KeeperTestSuite struct {
 	suite.Suite
+	helper apptesting.KeeperTestHelper
```

Comment:
Can we remove the embedded `suite.Suite` and just make the `apptesting.KeeperTestHelper` embedded here?

Reply:
(or try how that makes things)

Reply:
gotcha, I like that approach as well!